test/functional/multifactor_auths_controller_test.rb

```diff
@@ -10,21 +10,87 @@ class MultifactorAuthsControllerTest < ActionController::TestCase
       @request.cookies[:mfa_feature] = "true"
     end
 
-    context "when mfa enabled" do
+    context "when totp is enabled" do
       setup do
         @user.enable_totp!(ROTP::Base32.random_base32, :ui_only)
       end
 
-      context "on GET to new mfa" do
+      context "on calling delete to destroy" do
```

Comment:
We should put these test cases under the same context for better organization.

```
context "on DELETE to destroy"
  context "with correct otp"
  context "with incorrect otp"
```

We should also add destroy tests for with only webauthn is enabled (will catch the require mfa disabled case I mentioned before).

Reply:
Changed in https://github.com/rubygems/rubygems.org/pull/3821/commits/e9130e5742dd20587e31bea390bb88da28fda8ee